src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -150,6 +150,9 @@ namespace DataHVACGlobals {
     // OA Controller Heat Recovery Bypass Control Types
     int constexpr BypassWhenWithinEconomizerLimits(0);   // heat recovery controlled by economizer limits
     int constexpr BypassWhenOAFlowGreaterThanMinimum(1); // heat recovery ON at minimum OA in economizer mode
+    // OA Controller Economizer Staging
```

Comment:
Make these into an enum and use `getEnumerationValue` (the pattern is in 200+ places already).